programs/management/commands/quick_start.py

```diff
@@ -85,7 +84,7 @@ class Command(BaseCommand):
         {'abbr': 'chs', 'external': 'chs'},
         {'abbr': 'cccap', 'external': 'cccap'},
         {'abbr': 'chp', 'external': 'chp'},
-        {'abbr': 'coeitc', 'external': 'coexeitc'},
+        {'abbr': 'coexeitc', 'external': 'coexeitc'},
```

Comment:
The reason that we have a `name_abbreviated` and an `external_name` is because of this program. The `name_abbreviated` refers to which calculator to use. The `external_name` helps to keep translations in sync when you import translations. COEITC or the Colorado Earned Income Tax Credit has the same calculator as COEXEITC or the Colorado Expanded Earned Income Tax Credit. They have different descriptions because one is for citizens, and the other is for non-citizens. Therefore, `coexeitc` does not have a calculator, and will throw an error. I think that it is better for each program to have its own calculator, so could you add `'coexeitc': tax.Coeitc` to `co_tax_unit_calculators` in `programs/programs/co/pe/__init__.py`? FYI I merged my state folder pr into dev, so you will need to pull it to get that file.

Reply:
Oh thanks for the clarification. After adding coexeitc, do not see any errors so thanks for the recommended fix.